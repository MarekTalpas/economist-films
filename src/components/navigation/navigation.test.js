// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Nav from './navigation';

test('Nav renders correctly', () => {
  const tree : string = renderer.create(<Nav />).toJSON();
  expect(tree).toMatchSnapshot();
});
