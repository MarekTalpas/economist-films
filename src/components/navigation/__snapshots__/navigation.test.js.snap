exports[`test Nav renders correctly 1`] = `
<div>
  <ul>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Home
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        about
      </a>
    </li>
  </ul>
</div>
`;
